var/spack/repos/builtin/packages/cub/package.py

```diff
@@ -11,9 +11,14 @@ class Cub(Package):
     and other utilities for CUDA kernel programming."""
 
     homepage = "https://nvlabs.github.com/cub"
-    url      = "https://github.com/NVlabs/cub/archive/1.7.1.zip"
-    git      = "https://github.com/NVlabs/cub.git"
+    url      = "https://github.com/NVIDIA/cub/archive/1.12.0.zip"
+    git      = "https://github.com/NVIDIA/cub.git"
 
+    version('1.12.0',  sha256='92f0f39235db787f4850d5c4ffdaa7d5367a2f153bdd3b4a1161a6a2fefb7bfc')
+    version('1.12.0-rc0',  sha256='c9470cb5a23849e3143a3b2f07fe4d48ed3c8e0ec862b4d4c02ec15afb4fc331')
```

Comment:
Same comment as in https://github.com/spack/spack/pull/23364#discussion_r625114765

Reply:
Thank you for your review!

I'll remove `-rc` tags, but for `-1` tags of cub, could we just keep them?
Link: https://github.com/NVIDIA/cub/releases/tag/1.9.8-1